src/App.scss

```diff
@@ -79,3 +83,57 @@ $number-of-channels: 3;
     }
   }
 }
+
+#uploadIframe {
```

Comment:
Any particular reason for using IDs rather than classes?

Reply:
Erm, because I was only expecting one of them?